oggm/tests/test_prepro.py

```diff
@@ -1950,7 +1950,7 @@ def test_geodetic_mb_calibration_single_fl(self):
         assert 5 < mb_new.mu_star < 500
 
         # Check that inversion works
-        climate.apparent_mb_from_any_mb(gdir, mb_years=np.(1953, 2003, 1))
+        climate.apparent_mb_from_any_mb(gdir, mb_years=np(1953, 2003, 1))
```

Comment:
you want this to be `np.arange`

Reply:
> you want this to be `np.arange`

I've changed them.

But 2 `mb_years` in `test_find_calving_any_mb` get just 1 year (which is 2000), I have no idea about this.
Maybe it's related to this line `mb = massbalance.LinearMassBalance(ela_h=2000)`? What should we do for this?